src/components/GeneralInfo/GeneralInformation/GeneralInformation.js

```diff
@@ -66,31 +67,35 @@ class GeneralInformation extends Component {
             BiosCardWrapper,
             InfrastructureCardWrapper,
             ConfigurationCardWrapper,
-            CollectionCardWrapper,
+            SystemStatusCardWrapper,
+            DataCollectorsCardWrapper,
             children
         } = this.props;
         const Wrapper = store ? Provider : Fragment;
         return (
             <Wrapper {...(store && { store })}>
                 <div className="ins-c-general-information">
-                    <Grid sm={ 12 } md={ 6 } hasGutter>
+                    <Grid sm={12} md={6} hasGutter>
                         {SystemCardWrapper && <GridItem>
                             <SystemCardWrapper handleClick={ this.handleModalToggle } writePermissions={writePermissions} />
                         </GridItem>}
                         {OperatingSystemCardWrapper && <GridItem>
                             <OperatingSystemCardWrapper handleClick={ this.handleModalToggle } />
                         </GridItem>}
+                        {InfrastructureCardWrapper && <GridItem>
+                            <InfrastructureCardWrapper handleClick={ this.handleModalToggle } />
+                        </GridItem>}
                         {BiosCardWrapper && <GridItem>
                             <BiosCardWrapper handleClick={ this.handleModalToggle } />
                         </GridItem>}
-                        {InfrastructureCardWrapper && <GridItem>
-                            <InfrastructureCardWrapper handleClick={ this.handleModalToggle } />
+                        {SystemStatusCardWrapper && <GridItem>
+                            <SystemStatusCardWrapper handleClick={ this.handleModalToggle } />
                         </GridItem>}
                         {ConfigurationCardWrapper && <GridItem>
                             <ConfigurationCardWrapper handleClick={ this.handleModalToggle } />
                         </GridItem>}
-                        {CollectionCardWrapper && <GridItem>
```

Comment:
Let's not remove this wrapper so Edge team can keep it there and start using it once they choose to.